plugins/eos/gs-plugin-eos.c

```diff
@@ -779,6 +781,7 @@ gs_plugin_initialize (GsPlugin *plugin)
 	/* we already deal with apps that need to be proxied, so let's impede
 	 * the other plugin from running */
 	gs_plugin_add_rule (plugin, GS_PLUGIN_RULE_CONFLICTS, "generic-updates");
+	priv->flatpaks = g_ptr_array_new_with_free_func ((GDestroyNotify) g_object_unref);
```

Comment:
I just noticed this doesn't get freed - see how it's [done in the Flatpak plugin](https://github.com/endlessm/gnome-software/blob/master/plugins/flatpak/gs-plugin-flatpak.c#L95) in `eos_plugin_destroy()`.